source/developers/references/developer_guide/process/FAQ-about-pull-requests.rst

```diff
@@ -0,0 +1,175 @@
+      :doc:`contributor` for requirements needed for the Product team to
+      properly review your contribution. Ideally, this would be done before code
+      is written.
+
+#. If checks need to be enabled, or a CLA form needs to be submitted, the
+   Community PMs will help get checks running and help with any questions
+   regarding the CLA
+
+   .. image:: /_images/devguide_workflow_approval.png
+
+   .. image:: /_images/devguide_failed_cla_check.png
+            
+#. If there are checks failing on your pull request, you'll need to look into
+   what's causing the issue. The Community PM can direct you to someone to help
+   if needed.
+
+   #. Note: please see :doc:`../../running_pr_tests` that outlines common issues
+      that pop up while running tests
+   
+#. Once everything is up and running, the Community PMs are available to help
+   with any questions that come up while product and/or engineering review is
+   in-progress
+
+#. If the pull request review process takes a bit longer than expected, you may
+   notice the following alerts pop up, and the Community Project Managers might
+   ask you to take some action:
+
+   #. **“This branch is out-of-date with the base branch”** which means you'll
+      need to rebase your changes.
+
+      .. image:: /_images/devguide_out_of_date_branch.png
+
+   #. **“This branch has conflicts that must be resolved”** which means someone
+      has changed information in the same file on the target branch of the PR
+      (which is usually the master). You will need to reconcile those changes
+      with your own.
+
+      .. image:: /_images/devguide_branch_conflicts.png
+
+#. When all checks are green, tests are passing, there are no conflicts, and the
+   work is ready for engineering review, the Community PMs will assign the
+   appropriate reviewer / maintainer to take a look
+
+   .. image:: /_images/devguide_passing_checks.png
+
+#. During review, you may receive feedback or change requests that require
+   action on your part.
+
+#. Once all feedback and changes have been incorporated, the pull request will
+   be able to be merged by the reviewer / maintainer.
+
+As pull requests move through the process, the Community Project Managers update
+the PR's status, and use labels to denote different aspects / states of the PR.
+This helps reviewers better understand what action(s) are needed, and by who.
+
+A breakdown of these statuses and labels can be found at
+:doc:`pull-request-statuses` for reference (though, PR authors are not
+responsible for updating PR status or labels).
+
+On the right-hand side of the PR, you'll be able to see the status of your pull
+request, the Community PM responsible for the PR, and the last date the
+Community PM checked in on it. To do this, find the "Projects" section, and
+click on the "⌄" caret in the top right corner to see a display like this:
+
+.. image:: /_images/devguide_pr_status.png
```

Comment:
@itsjeyd moving our discussion from Google Docs - can you take a look at this language? You can see the page rendered here: https://docsopenedxorg--395.org.readthedocs.build/en/395/developers/references/developer_guide/process/FAQ-about-pull-requests.html

Reply:
@sarina I checked the screenshot and paragraph above it - LGTM. Since PRs can be part of multiple projects, though, the text should probably mention the Contributions board specifically?

I.e., something like:

> On the right-hand side of the PR, you’ll be able to see the status of your pull request, the Community PM responsible for the PR, and the last date the Community PM checked in on it. To do this, find the “Projects” section, and click on the “⌄” caret ~~in the top right corner~~ next to "Contributions" to see a display like this: ...